app/src/main/java/com/github/multimatum_team/multimatum/LocalDateTimeDeserializer.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.multimatum_team.multimatum
```

Comment:
It would make sense to move this class to the `util` package.

Reply:
i thought about it but i wasn't sur. Thanks for the confirmation